src/common/localization/zh-ch.ts

```diff
@@ -0,0 +1,509 @@
+        securityToken: {
+            name: {
+                title: "名称", // Name
+            },
+            key: {
+                title: "键", // Key
+            },
+        },
+        securityTokens: {
+            title: "安全令牌", // Security Tokens
+            description: "安全令牌用于加密项目配置中的敏感数据",
+            // Security tokens are used to encrypt sensitive data within your project configuration
+        },
+        version: {
+            description: "版：", // Version:
+        },
+        commit: "提交SHA", // Commit SHA
+        devTools: {
+            description: "打开应用程序开发人员工具以帮助诊断问题", // Open application developer tools to help diagnose issues
+            button: "切换至开发人员工具", // Toggle Developer Tools
+        },
+        reload: {
+            description: "重新加载应用，放弃当前所有更改", // Reload the app discarding all current changes
+            button: "刷新应用", // Refresh Application
+        },
+        messages: {
+            saveSuccess: "成功保存应用程序设置", // Successfully saved application settings
+        },
+    },
+    projectSettings: {
+        title: "项目设定", // Project Settings
+        securityToken: {
+            title: "安全令牌", // Security Token
+            description: "用于加密项目文件中的敏感数据", // Used to encrypt sensitive data within project files
+        },
+        save: "保存项目", // Save Project
+        sourceConnection: {
+            title: "源连接", // Source Connection
+            description: "从何处加载素材", // Where to load assets from
+        },
+        targetConnection: {
+            title: "目标连接", // Target Connection
+            description: "在哪里保存项目和导出数据", // Where to save the project and exported data
+        },
+        videoSettings: {
+            title: "视频设定", // Video Settings
+            description: "提取帧以进行标记的速率", // The rate at which frames are extracted for tagging.
+            frameExtractionRate: "帧提取率（每视频每秒的帧数）", // Frame Extraction Rate (frames per a video second)
+        },
+        addConnection: "添加连接", // Add Connection
+        messages: {
+            saveSuccess: "成功保存${project.name}项目设置", // Successfully saved ${project.name} project settings
+        },
+    },
+    projectMetrics: {
+        title: "项目指标", // Project Metrics
+        assetsSectionTitle: "素材", // Assets
+        totalAssetCount: "素材总数", // Total Assets
+        visitedAssets: "已访问素材（${count}）", // Visited Assets (${count})
+        taggedAssets: "已标记素材（${count}）", // Tagged Assets (${count})
+        nonTaggedAssets: "未标记素材（${count}）", // Not Tagged Assets (${count})
+        nonVisitedAssets: "未访问素材（${count}）", // Not Visited Assets (${count})
+        tagsSectionTitle: "标记与标签", // Tags & Labels
+        totalRegionCount: "标记区域总数", // Total Tagged Regions
+        totalTagCount: "标签总数", // Total Tags
+        avgTagCountPerAsset: "每个素材的平均标签数", // Average tags per asset
+    },
+    tags: {
+        title: "标签", // Tags
+        placeholder: "新增标签", // Add new tag
+        editor: "标签编辑器", // Tags Editor
```

Comment:
changed to '编辑标签'. although in English translation there is the word 'editor', but '编辑器' does not exist anywhere else and not common to understand. thus changed to simpler phase as just '编辑标签'.

also changed ‘新增标签’ to ‘添加标签’.  (since 'add new connection' has been translated to '添加连接' instead of '新增连接').  moreover, '添加' is definitely verb while '新增' could be misunderstood as adj.